### PR TITLE
Deploy wso2-is helm charts in one go

### LINKED ISSUE #64: Obtain service account name from values.yaml in HELM Charts
**Description:**
The `service account name` can be obtained from the `values.yaml` which we request users to customize, than hardcoding the value `wso2svc-account` for it

**Suggested Labels:**
Type/Enhancement

**Suggested Assignees:**
Kasun Siyambalapitiya
**Affected Product Version:**
5.6.0 Helm Charts
**OS, DB, other environment details and versions:**    
N/A
**Steps to reproduce:**
N/A

**Related Issues:**
N/A

### LINKED ISSUE #65: Deploy wso2-is helm charts in one go
**Description:**
The current deployment needs to run two `helm install ` commands to deploy `wso2-is` in a kubernetes cluster. This needs to be changed so that `wso2-is` can be deployes from one `helm install` command.

**Suggested Labels:**
Type/Enhancement
**Suggested Assignees:**
Kasun Siyambalapitiya

**Affected Product Version:**
5.6.0
**OS, DB, other environment details and versions:**    
N/A
**Steps to reproduce:**
N/A

**Related Issues:**
N/A

### PR DESCRIPTION
**Pull request by kasunsiyambalapitiya**
Resolves #65 
Resolves #64 

## Automation tests
 - Unit tests 
   > N/A
 - Integration tests
   > N/A
## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? N/A
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: helm/is/README.md [hunk 1]
```diff
@@ -72,20 +72,8 @@ b. Open the `<HELM_HOME>/is-conf/values.yaml` and provide the following values.
 | `sharedDeploymentLocationPath`  | NFS shared deployment directory(`<IS_HOME>/repository/deployment`) location for EI        |
```
Comment (DilanUA) [inline helm/is/README.md]: This should be IS, I believe, :)

File: helm/is/is/Chart.yaml [hunk 2]
```diff
@@ -15,5 +15,5 @@
 apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart for the deployment of WSO2 IS Deployment
-name: is-deployment
+name: is
 version: 1.0.0
```
Comment (DilanUA) [inline helm/is/is/Chart.yaml]: Shall we check if this version corresponds to something helm charts specific or it is some dumb value that we provide, if it's something that we provide, we may rename this to the upcoming repo release, WDYT?

File: helm/is/README.md (hunk 1 above)
Comment (kasunsiyambalapitiya) [reply helm/is/README.md]: Fixed in 5908da3

File: helm/is/is/Chart.yaml (hunk 2 above)
Comment (kasunsiyambalapitiya) [reply helm/is/is/Chart.yaml]: Added the relevant product version for above in 5908da3

**Comment by kasunsiyambalapitiya** [comment]
Sent the changes with a different PR to branch `5.6.x` as this PR contain additional commits resulted due to branch comparing

**Pull request closed without merging**